wagn/README.rdoc

```diff
@@ -0,0 +1,104 @@
+=Wagn: how pioneers roll
+
+==Basics
+
+Wagn is serious web development made fun.
+
+"Wagneers" have used Wagn to create open community sites, private knowledge management sites, public proposal submission sites with private back-ends for review, project management systems, wikis, blogs, journals, forums, and more.
+
+Install Wagn, open a webpage, and get started. With Wagn's wiki-inspired building blocks, you can collaborate from day one on content, design, and structure create elegant web systems on the fly.  And ruby developers can take these creations even further with Wagn's development framework. Wagn's innovative Mods API supports boundless creativity while integrating seamlessly with structures created on the site.
+
+Try it out with automated setup at Cloudstore[https://cldstr.com/wagn.org/wagn] (free hosting for one trial month) or set it up yourself following the instructions below.
+
+
+==System Requirements
+
+Ruby[http://www.ruby-lang.org/en/]:: version 1.9.3 or higher
```

Comment:
I think it is good to indicate a preferred version.  Maybe too much detail, but the rails > 4 is also significant, no?